snap/pack/pack.go

```diff
@@ -250,7 +250,8 @@ func prepare(sourceDir, targetDir, buildDir string) (snapName string, err error)
 	}
 
 	// build the package
-	snapName = fmt.Sprintf("%s_%s_%v.snap", info.Name(), info.Version, debArchitecture(info))
+	// TODO parallel-install: use of proper instance/store name
+	snapName = fmt.Sprintf("%s_%s_%v.snap", info.InstanceName(), info.Version, debArchitecture(info))
```

Comment:
this could be StoreName as well, (we have a couple other places where that is true), under the assumption that at some point in the process we might again s/InstanceName/Name/, is probably best to use InstanceName, not sure we need the todo marker

Reply:
Ok, dropping TODO for now.